di_website/templates/includes/partials/multi_authors.html

```diff
@@ -0,0 +1,14 @@
+<div class="m-authors">
+    <h3 class="m-authors__title">Written by</h3>
+    <p class="m-authors__copy">
+      {% for author in authors %}
+
+        {% if forloop.counter < author_count %}
+          {{ author.value.name }},
+        {% else %}
+          {{ author.value.name }}
+        {% endif %}
+
+      {% endfor %}
+    </p>
+</div>
```

Comment:
We now have two templates in the same folder, one named `multi-authors`, and this one named `multi_authors`... I find that a tad redundant. Unless they're too different, get rid of the duplicate ... otherwise, rename it to something more distinctive, like `multi-authors-card`

Reply:
The previous template does the job with minimal changes to the html